src/alert.js

```diff
@@ -1,15 +1,15 @@
-import React from 'react'
+import React, { forwardRef } from 'react'
 import PropTypes from 'prop-types'
 import MUIAlert from '@material-ui/lab/Alert'
 import AlertTitle from '@material-ui/lab/AlertTitle'
 
-const Alert = ({ severity, message, header, width, ...props }) => {
+const Alert = forwardRef(({ severity, message, header, width, ...props }, ref) => {
```

Comment:
why dont we add forwardRef at the end where components are exported for a cleaner look?
```
export default forwardRef(Alert);
```

Reply:
Proptype warning

<img width="687" alt="Screen Shot 2021-04-29 at 6 45 54 PM" src="https://user-images.githubusercontent.com/46459367/116627439-241ca400-a91b-11eb-950b-38560fe4545d.png">

Reply:
that's interesting 🤔